### ISSUE TITLE
Completed Tasks not going to Completed

### ISSUE THREAD

**Issue by jayavman** [humhub/tasks#205]
Hi, 

When tasks are completed they aren't going to the completed folder. Is this normal? If so how do you assign it to the completed folder?

![Screen Shot 2022-10-04 at 3 32 05 pm](https://user-images.githubusercontent.com/114973646/193734873-02625e15-e0a1-429f-8e21-949699ecfbe7.png)

**Comment by luke-** [comment]
This should only include completed task lists (other than the main list).

@yurabakhtin Can you confirm that? If so, we should rename "Completed" to "Completed Lists" and hide it when empty?

**Comment by jayavman** [comment]
Not sure what you mean by "only include completed task lists" 

You have to assign a task to a Task List. No option to not have it in a task list. Also they do not have an option to put it in a Completed List.
<img width="629" alt="Screen Shot 2022-10-05 at 10 31 06 am" src="https://user-images.githubusercontent.com/114973646/193949189-cc0382a1-7096-4a7c-97c3-215c2bd95d01.png">

**Comment by jayavman** [comment]
I think I made things worse, I then added a new Task List called Completed. After I did that and assigned that job from Unsorted to Completed. It then went to the completed folder but looks like the below.
Missing title and everything now, Its like the job has completely disappeared!
<img width="1092" alt="Screen Shot 2022-10-05 at 10 33 58 am" src="https://user-images.githubusercontent.com/114973646/193949451-ce6e061f-5e2b-49a0-b75c-36bd7163f166.png">
